Gaijinizer patch/Map036.txt

```diff
@@ -108,108 +110,118 @@ you need any supplements ☆#MTLed
 
 寝起きの一言がソレって、どんな夢見てたんだよ…
 > CONTEXT: Map036/events/13/pages/2/8/Dialogue/other2
-That's ............ What a dream I woke up to
-sleep...what a dream ...#MTLed
+Hey now...
+
+You must've had one hell of a dream to wake up
+saying that sort of stuff...#Ron
 > END STRING
 
 > BEGIN STRING
 んーーー……それがな………
 
 実際見ていた俺でもちょっと信じられなかったんだが……
 > CONTEXT: Map036/events/13/pages/2/23/Dialogue/other2
-Hmm...that's...... Even though I was actually
-watching, I couldn't believe for a moment ...#MTLed
+Huuh... That's...
+
+It was pretty unbelievable, even for me...#Ron
 > END STRING
 
 > BEGIN STRING
 この休憩室までお前を運んだの
 
 そのアイアンメイデン本人なんだよ……。
 > CONTEXT: Map036/events/13/pages/2/27/Dialogue/other2
-I took you to this break room. 
-I'm the Iron Maiden himself ....#MTLed
+The one who carried you back to this break room
+was none other than Iron Maiden herself...#Ron
 > END STRING
 
 > BEGIN STRING
 えっ…！？
 
 な、なんで…！？
 > CONTEXT: Map036/events/13/pages/2/31/Dialogue/kazuya
-eh…! ? Why ...! ?#MTLed
+Huh...!?
+
+W- why...!?#Ron
 > END STRING
 
 > BEGIN STRING
 だから言ってるじゃねぇか、俺も信じられんのよ。
 
 あの時お前は、力尽きて確実に気を失ってた。
 > CONTEXT: Map036/events/13/pages/2/35/Dialogue/other2
-So you're not saying, I can't believe it.
-At that time you were completely out of your
-mind.#MTLed
+That's what I'm saying, I can't believe it either.
+At the time your strength had completely drained
+and you were unconcious.#Ron
 > END STRING
 
 > BEGIN STRING
 俺も、お前はあのままアイアンメイデンに喰われると思って
 
 自分のねぐらに帰る準備してたんだけどさ…
 > CONTEXT: Map036/events/13/pages/2/39/Dialogue/other2
-I also think that you will be covered by Iron
-Maiden as it is. I was preparing to go back to my
-own roost ...#MTLed
+I thought for sure Iron Maiden was gonna consume
+you. I was just about ready to go back home to my
+pad...#Ron
 > END STRING
 
 > BEGIN STRING
 どういうわけかアイツ、お前を取り込まずに抱えて
 
 この休憩室まで静かに運んでったんだぜ。
 > CONTEXT: Map036/events/13/pages/2/43/Dialogue/other2
-Somehow, he's holding you without taking you
-I was carrying it quietly to this break room.#MTLed
+For some reason, she held you without consuming
+you, and quietly carried you here to this break
+room.#Ron
 > END STRING
 
 > BEGIN STRING
 しかも、フツーにこの部屋入ってきたって事は
 
 『お前を襲う意志』がアイツにあの時点で無かったって事だ
 > CONTEXT: Map036/events/13/pages/2/47/Dialogue/other1
-What's more, this room has entered nature
-"The will to attack you" was not at that point in
-the guy's#MTLed
+What's more, the fact she could casually stroll in
+here means she had no intention to attack you.#Ron
 > END STRING
 
 > BEGIN STRING
 全然分からん……。
 
 あいつは、お前を襲うための兵器じゃなかったのか？
 > CONTEXT: Map036/events/13/pages/2/51/Dialogue/other2
-I do not know at all .... Wasn't that guy a
-weapon to attack you?#MTLed
+I don't get it at all...
+
+Wasn't she supposed to be a weapon made to attack
+you?#Ron
 > END STRING
```

Comment:
We already used "attack" in the previous line, so let's change the instance of "attack" here to "dispose". It also fits in more thematically.